org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java

```diff
@@ -42,13 +42,15 @@
 
   public static String SonarConsoleRemoveAction_tooltip;
 
-  public static String SonarDebugOutputAction_tooltip;
-
   public static String SonarShowConsoleAction_tooltip;
   public static String SonarShowConsoleAction_never_text;
   public static String SonarShowConsoleAction_onOutput_text;
   public static String SonarShowConsoleAction_onError_text;
 
+  public static String ConfigureLoggingAction_tooltip;
+  public static String ConfigureLoggingAction_verbose_text;
+  public static String ConfigureLoggingAction_analyzer_text;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "ConfigureLoggingAction_analyzer_text" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS3008)